src/app/components/Map/Map.tsx

```diff
@@ -77,41 +89,50 @@ export default function MapBox(): JSX.Element {
       },
     }
 
-    const pointData = {
+    const pointData: GeoJSON.FeatureCollection<GeoJSON.Geometry> = {
       type: 'FeatureCollection',
       features: [
         {
           type: 'Feature',
           properties: {},
           geometry: {
             type: 'Point',
-            coordinates: location1,
+            coordinates: location1 as GeoJSON.Position,
```

Comment:
I don't think you need to cast `location1` as this specific type, because it already is of this type :)

Reply:
didnt work : Type 'Position | null' is not assignable to type 'Position'.